platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -0,0 +1,204 @@
+    String profileName;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProfileInstallCommand.class);
+
+    @Override
+    protected void doExecute(ApplicationService applicationService,
+                             FeaturesService featuresService, BundleService bundleService) {
+
+        Optional<Feature> optionalProfile = getProfile(applicationService, profileName);
+        Optional<Map<String, List<String>>> optionalDevProfile = getDevProfile(profileName);
+
+        if (!optionalProfile.isPresent() && showDevProfiles) {
+            if (optionalDevProfile.isPresent()) {
+                Map<String, List<String>> profile = optionalDevProfile.get();
+                startApps(applicationService, profile.get(DEV_PROFILE_START_APPS));
+                installFeatures(featuresService, profile.get(DEV_PROFILE_INSTALL_FEATURES));
+                uninstallFeatures(featuresService, profile.get(DEV_PROFILE_UNINSTALL_FEATURES));
+                stopBundles(bundleService, profile.get(DEV_PROFILE_STOP_BUNDLES));
+                uninstallInstallerModule(featuresService);
+            } else {
+                printError(String.format("Development Profile: %s not found", profileName));
+            }
+        } else {
+            if (optionalProfile.isPresent()) {
+                List<String> profileApps = optionalProfile.get().getDependencies().stream()
+                        .map(Dependency::getName)
+                        .collect(Collectors.toList());
+                startApps(applicationService, profileApps);
+                uninstallInstallerModule(featuresService);
+            } else {
+                printError(String.format("Profile: %s not found", profileName));
+            }
+        }
+    }
+
+    private void startApps(ApplicationService applicationService, List<String> startupApps) {
+        if (startupApps != null) {
+            printSectionHeading("Starting Applications");
+            startupApps.forEach(application -> {
+                printItemStatusPending("Starting: ", application);
+                boolean installSuccess = executeAsSystem(() -> {
+                    try {
+                        applicationService.startApplication(application);
+                        return true;
+                    } catch (ApplicationServiceException e) {
+                        printItemStatusFailure("Start Failed: ", application);
+                        LOGGER.error(String.format("An error occurred starting application: %s", application), e);
+                        return false;
+                    }
+                });
+                if (installSuccess) {
+                    printItemStatusSuccess("Started: ", application);
+                } else {
+                    return;
+                }
+            });
+        }
+    }
+
+    private void uninstallInstallerModule(FeaturesService featuresService) {
+        try {
```

Comment:
How about one console trace indicating you are finalizing installation (kind of like done on the UI at this stage)?